__tests__/news.test.js

```diff
@@ -48,3 +48,51 @@ describe("/api/topics", () => {
     });
   });
 });
+
+//Test 3 - Get all articles
+describe("/api/articles/", () => {
+  describe("GET Article object", () => {
```

Comment:
get all articles is ticket 8, i know it seems harsh but we want to be tackling one ticket at a time so let's get rid of this for now